app/config/services.php

```diff
@@ -146,3 +329,53 @@
 $di->set('acl', function () {
     return new Acl();
 });
+
+
+/*
+ *  Translation application use gettext
+ **/
+$di->set(
+    'translation',
+    function () use ($di) {
+        $translation = new Gettext(
+            [
+                'locale' => $di->get('config')->language,
+                'directory' => ROOT_DIR . '/apps/lang',
+                'defaultDomain' => 'messages',
+
+            ]
+        );
+        return $translation;
+    },
+    true
+);
+
+/**
+ * Translation function call anywhere
+ *
+ * @param $string
+ *
+ * @return mixed
+ */
+function t($string)
+{
+    $translation = DI::getDefault()->get('translation');
+    return $translation->_($string);
+}
+
+/*
+ *  Phalcon Debugger
+ **/
+if ($config->application->debug) {
+    (new \Phalcon\Debug)->listen();
+
+    function dpm($object, $kill = true)
+    {
+        echo '<pre style="text-aling:left">';
+        print_r($object);
+        if ($kill) {
+            die('END');
```

Comment:
@sergeyklay I know this PR is closed, but WHY ? Why would you ask to write that `$kill && die('END')` on the same line ? the way he wrote this if statement makes it very readable and easy to understand for everyone. 

And from my perspective, this `dpm()` function should not be here at all. every developer should write it's own debug functions because usually they are used only in development mode.  Symfony has a repository for this purpose `symfony/var-dumper`. The same thing could be done for phalcon as well (sorry if there is one and I don't know about it)